client/src/stories/SelectField.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SelectField from '../components/SelectField.js';
+import InputDropdown from '../components/InputDropdown';
+
+export default {
+  title: 'Select field',
+};
+
+export function Input() {
+  return (
+    <SelectField>
+      <InputDropdown placeholder="Projekttyp" />
+    </SelectField>
```

Comment:
`<InputDropdown placeholder="Projekttyp" />` is never rendered because `SelectField` doesn't expect a `children`.
This is not working like expected.